runner/lib/helpers/buffer-line-transform.d.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef,no-unused-vars */
+
+/// <reference types="node" />
+
+import { Transform, TransformOptions } from 'stream';
+
+export interface BufferLineTransformOptions extends TransformOptions {
+  /** line break matcher for Buffer.indexOf() (default: 10 ) */
+  break?: Buffer | string | number;
```

Comment:
It will help considerably in portability to use the common ArrayBuffer or Uint8Array types.